.gitignore

```diff
@@ -10,6 +10,7 @@ build_android/.*
 build_android/*.apk
 build_android/Play.iml
 build_android/projectFilesBackup/
+build_ios/build
```

Comment:
Do we need to ignore that directory? The build folder for iOS was removed a while ago now that CMake is used to generate the project. I might be wrong, but there's probably no references to "build_ios" anywhere in the project at the moment.